packages/SwingSet/src/liveslots/virtualObjectManager.js

```diff
@@ -544,61 +632,53 @@ export function makeVirtualObjectManager(
           },
         });
       }
-      harden(wrappedState);
+      harden(state);
 
       if (initializing) {
         cache.remember(innerSelf);
       }
-      const self = actualize(wrappedState);
       let toHold;
       let toExpose;
-      const facetiousness = assessFacetiousness(self);
-      switch (facetiousness) {
-        case 'one': {
-          toHold = Far(tag, self);
-          vrm.checkOrAcquireFacetNames(kindID, null);
-          toExpose = toHold;
-          break;
-        }
-        case 'many': {
-          toExpose = {};
-          toHold = [];
-          const facetNames = Object.getOwnPropertyNames(self).sort();
-          assert(
-            facetNames.length > 1,
-            'a multi-facet object must have multiple facets',
-          );
-          vrm.checkOrAcquireFacetNames(kindID, facetNames);
-          for (const facetName of facetNames) {
-            const facet = Far(`${tag} ${facetName}`, self[facetName]);
-            toExpose[facetName] = facet;
-            toHold.push(facet);
-            facetToCohort.set(facet, toHold);
-          }
-          harden(toExpose);
-          break;
+      unweakable.add(state);
+      if (facetNames === null) {
+        const context = { state };
+        unweakable.add(context);
```

Comment:
oh, crud, I got the comment backwards, and it's going to confuse me in its current state

please replace with:

`'context' does not need a linkToCohort because it holds the facets (which hold the cohort)`

sorry